### DISCUSSION TITLE
Footer

### DISCUSSION THREAD

**Discussion by Shalini469717** [EddieHubCommunity/LinkFree#5930] [Ideas]
![image](https://user-images.githubusercontent.com/99305545/230731700-4a54e814-67bd-4371-a363-74fda0beec95.png)
Would it be good to add some kind of footer or maybe go to top button, because I have to scroll to the top to go the github button.

**Comment by harisdev-netizen** [comment]
Having a footer or a "go to top" button can improve the user experience for your website visitors. A footer can provide additional navigation options and important information, such as contact details or copyright notices. A "go to top" button can save users the hassle of scrolling back up to the top of the page.

**Reply by Shalini469717** [reply]
Yeah I was wondering about the same while scrolling through the website

**Reply by harisdev-netizen** [reply]
Let's hope someone would work on this enhancement feature.
